client/modules/food/components/packing/Packages.js

```diff
@@ -30,6 +32,12 @@ class Packages extends Component {
     this.props.load()
   }
 
+  componentWillReceiveProps(nextProps) {
+    if (this.props.packSaving !== nextProps.packSaving) {
+      this.props.load()
```

Comment:
You don't need to reload if you send the data back in the right format, the api middleware will handle updating the store. See below